test/react-markdown.test.js

```diff
@@ -8,7 +8,8 @@ const visit = require('unist-util-visit')
 const ReactDom = require('react-dom/server')
 const renderer = require('react-test-renderer')
 const math = require('remark-math')
-const reactKatex = require('react-katex')
+const TeX = require('@matejmazur/react-katex')
+const {render} = require('@testing-library/react')
```

Comment:
with how `@matejmazur/react-katex` renders, it needs a browser-like DOM document to render into.
testing library react's render method provides a clean way of handling this.